people/asheesh-agarwal.md

```diff
@@ -0,0 +1,5 @@
+![Asheesh Agarwal](photos/Asheesh\ Agarwal.jpg)
```

Comment:
Please rename the JPG file to asheesh-agrawal.jpg, consistent with the other names.